airflow/models/baseoperator.py

```diff
@@ -1184,7 +1197,29 @@ def render_template_fields(
         """
         if not jinja_env:
             jinja_env = self.get_template_env()
-        self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
+
+        if self.template_all_fields:
+            templatable_fields = (
```

Comment:
Was wondering if anyone has any comment on how I did this. Goal was to get attributes that belong only to child classes of `BaseOperator`, would like to know if this approach is satisfactory, or if there is a better way to accomplish it.